client/components/language-picker/modal.tsx

```diff
@@ -144,6 +135,21 @@ const LanguagePickerModal: React.FC< Props > = ( {
 							</span>
 						</FormLabel>
 					) }
+					{ selectedLanguage.isTranslatedIncompletely && (
```

Comment:
The `selectedLanguage.isTranslatedIncompletely` seems redundant as it's already [one of the condition to render the parent element](https://github.com/Automattic/wp-calypso/pull/92772/files#diff-e2c927e8e979f0f2d70ee47e9b25071d8e3232a3c9a1842993925b8dfb5c8c11L125).

Reply:
Good catch!